docs/ops/arithmetic/Tanh_1.md

```diff
@@ -6,15 +6,21 @@
 
 **Short description**: Tanh element-wise activation function.
 
-**Attributes**: has no attributes
+**Attributes**:
+
+    No attributes available.
 
 **Inputs**:
 
-*   **1**: Input tensor x of any floating point type. Required.
+* **1**: A tensor of type T. **Required.**
```

Comment:
Mention that the input shape is arbitrary. Refer to https://github.com/openvinotoolkit/openvino/pull/5369